src/ucs/config/global_opts.h

```diff
@@ -156,6 +157,11 @@ ucs_status_t ucs_global_opts_clone(void *dst);
 void ucs_global_opts_release();
 void ucs_global_opts_print(FILE *stream, ucs_config_print_flags_t print_flags);
 
+void ucs_init_ucm_opts();
+void ucs_opts_cleanup();
```

Comment:
can it be part of ucs_global_opts_init/cleanup?

Reply:
technically yes, but this is really bad idea: have to move from ucm_opts.c into global_opts.c set of macro(table prefix, datatypes), declare ucm_global_config_table as public...

Reply:
so maybe add ucm_opts.h?

Reply:
> so maybe add ucm_opts.h?

done